Source: http://to.ly/cJB6
Upstream: Yes - fixed in trunk.
Reason: make mod_dav_svn compile with apache 2.4.
--- a/subversion/mod_dav_svn/util.c    (revision 1299310)
+++ b/subversion/mod_dav_svn/util.c    (working copy)
@@ -627,8 +627,14 @@
         if (errscan->desc == NULL)
             continue;
 
-        if (errscan->save_errno != 0) {
+#if AP_MODULE_MAGIC_AT_LEAST(20091119,0)            
+        if (errscan->aprerr != 0) {
+            errno = errscan->aprerr;
+#else
+        if (errscan->save_errno != 0) {
             errno = errscan->save_errno;
+#endif
+
             ap_log_rerror(APLOG_MARK, level, errno, r, "%s  [%d, #%d]",
                           errscan->desc, errscan->status, errscan->error_id);
         }
